src/main/java/thecodewarrior/hooked/client/KeyBinds.kt

```diff
@@ -40,7 +40,8 @@ object KeyBinds {
     }
 
     @SubscribeEvent
-    fun onKeyInput(e: InputEvent.KeyInputEvent) {
+    @Suppress("UNUSED_PARAMETER") // we need the parameter to conform to the event signature, but we don't use it
+    fun onKeyInput(unused: InputEvent.KeyInputEvent) {
```

Comment:
I don't think the comment is really necessary here. Anyone who knows anything about Forge modding will know why we still need the parameter. Just suppressing the warning should be good enough.